documentation/docs/json/libraries.md

```diff
@@ -113,6 +109,38 @@ val transformed = stream.through(codec.transformOpt(f1, (i: Int) => (i > 0).guar
 transformed.compile.to(collector.pretty())
 ```
 
+#### Migrating from `circe-fs2`
+
+If you were using [`circe-fs2`][circe-fs2] to emit streams of `Json` values, you can easily switch to `fs2-data-json-circe`. Just replace your usages of `stringStreamParser` or `byteStreamParser` by usage of `fs2.data.json.ast.parse`.
+
+For instance if you had this code:
+
+```scala mdoc:nest
+import io.circe.fs2._
+
+import cats.effect._
+
+input[SyncIO]
+  .through(stringStreamParser)
+  .map(_.spaces2)
+  .compile
+  .toList
+  .unsafeRunSync()
+```
+
+You can replace it by
+
+```scala mdoc:nest
+import fs2.data.json._
+import fs2.data.json.circe._
+
+input[Fallible]
+  .through(ast.parse)
+  .map(_.spaces2)
+  .compile
+  .toList
+```
+
```

Comment:
Maybe we should add a reference to the docs about `CharLikeChunks` here so people that were using `byteStreamParser` know what to import?